cohere_aws/client.py

```diff
@@ -747,6 +747,72 @@ def create_finetune(
         bucket, old_short_key = parse_s3_url(s3_models_dir + job_name)
         s3_resource.Bucket(bucket).objects.filter(Prefix=old_short_key).delete()
 
+    def export_finetune(
+        self,
+        name: str,
+        s3_merged_weights_dir: str,
+        s3_tensorrtllm_engine_dir: str,
+        arn: str,
+        instance_type: str = "ml.p4de.24xlarge",
+        role: Optional[str] = None,
+    ) -> None:
+        """Export the merged weights to the TensorRT-LLM inference engine.
+
+        Args:
+        name (str): The name to give to the export task.
+        s3_merged_weights_dir (str): An S3 path pointing to the directory of the merged weights.
+        s3_tensorrtllm_engine_dir (str): An S3 path pointing to the directory where the TensorRT-LLM engine will be saved.
+        arn (str): The product ARN of the bring your own finetuning algorithm.
+        instance_type (str, optional): The EC2 instance type to use for export. Defaults to "ml.p4de.24xlarge".
+        role (str, optional): The IAM role to use for export.
+            If not provided, sagemaker.get_execution_role() will be used to get the role.
+            This should work when one uses the client inside SageMaker. If this errors out,
+            the default role "ServiceRoleSagemaker" will be used, which generally works outside SageMaker.
+        """
+        if name == "model":
```

Comment:
Why prevent the name from being "model"?

Reply:
I copied this line from https://github.com/cohere-ai/cohere-aws/blame/main/cohere_aws/client.py#L701, which I guess it is because `model.tar.gz` is some reserved name for our internal use, e.g., here, but I am not sure. 

Probably removing it won't cause issues, but putting it there is not harmful, so I decided to put it here. I can remove this if you want

Reply:
Ah, I think its fine to keep for now 😄